core/trino-main/src/main/java/io/trino/sql/planner/planprinter/anonymize/WindowNodeRepresentation.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.sql.planner.planprinter.anonymize;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.trino.sql.planner.Symbol;
+import io.trino.sql.planner.plan.PlanNodeId;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+import static io.trino.sql.planner.plan.WindowNode.Specification;
+import static java.util.Objects.requireNonNull;
+
+public class WindowNodeRepresentation
+        extends AnonymizedNodeRepresentation
+{
+    private final Specification specification;
```

Comment:
Don't rely on Specification from WindowNode directly, extract `partitionBy` and `orderings` from it